docs/src/atoms/dropdown.js

```diff
@@ -15,8 +13,8 @@ import React from 'react';
 import TableProps from '../global/tableProps';
 
 const getImageUrl = (imageFileName) => window.location.host.indexOf('localhost') > -1 ?
-    `/_/docs/src/images/${imageFileName}` :
-    `/images/${imageFileName}`;
+    `/global/images/${imageFileName}` :
+    `/global/images/${imageFileName}`;
```

Comment:
If this silliness is no longer necessary, we can get rid of it entirely.
It now looks like the path is the same in either case.
This was hear because we had to do that strange-looking path for localhost, but not for the deployed UI Docs website.  If that's no longer the case, let's kill it.